subprojects/docs/src/docs/userguide/running-builds/configuration_cache.adoc

```diff
@@ -956,6 +956,17 @@ include::sample[dir="snippets/valueProviders/fileContentsDo/kotlin",files="build
 In general, you should avoid reading files at configuration time, to avoid invalidating configuration cache entries when the file content changes.
 Instead, you can connect the `Provider` returned by link:{javadocPath}/org/gradle/api/provider/ProviderFactory.html#fileContents-org.gradle.api.file.RegularFile-[providers.fileContents()] to task properties.
 
+[[config_cache:requirements:safe_credentials]]
```

Comment:
@bamboo Preview: https://builds.gradle.org/repository/download/Gradle_Master_Check_BuildDistributions/54976304:id/distributions/gradle-7.7-docs.zip!/gradle-7.7-20220815195249%2B0000/docs/userguide/configuration_cache.html#config_cache:requirements:safe_credentials

Reply:
Nice! 👍

@nathan-contino please review this paragraph.

Reply:
Reviewed!

Reply:
Thanks, @nathan-contino! It's great to be able to count on your professional advice 👍